src/main/java/org/jboss/set/overview/ejb/Aider.java

```diff
@@ -385,4 +388,27 @@ public void updatePayloadData() {
         }
         return null;
     }
+
+    public static List<VersionUpgrade> getComponentUpgrades(String component, String tag1, String tag2) {
+        try {
+            StreamComponent streamComponent = aphrodite.getStream(EAP70ZSTREAM).getComponent(component);
+            return aphrodite.getRepository(streamComponent.getRepositoryURL().toURL()).getComponentUpgrades(tag1,tag2);
```

Comment:
possible NPE for invalid component parameter
```
	23:17:02,774 WARNING [org.jboss.set.overview.ejb.Aider] (default task-2) null: java.lang.NullPointerException
	at deployment.prbz-overview.war//org.jboss.set.overview.ejb.Aider.getComponentUpgrades(Aider.java:395)
	at deployment.prbz-overview.war//org.jboss.set.overview.ApiResource.getComponentUpgrades(ApiResource.java:109)
	at deployment.prbz-overview.war//org.jboss.set.overview.ApiResource$Proxy$_$$_WeldClientProxy.getComponentUpgrades(Unknown Source)
```

Reply:
Well yes but it gets handled, or do you want a better error message?

Reply:
Yes, it would be better, maybe just a new catch for NPE case with informative message ? 
I got an empty list and found NPE warning in server log when I hit http://localhost:8080/prbz-overview/rest/api/tags/wildfly-wildfly, Then I realized that that component does not exist in 7.0.z stream definition.